src/components/DatasetCard.vue

```diff
@@ -168,6 +168,7 @@ export default {
     propogateCardAction: function(action){
       EventBus.$emit("PopoverActionClick", action)
       this.$emit('contextUpdate', action)
```

Comment:
This should either be removed or replaced by` EventBus.$emit("contextUpdate", action) ` completely.